doc/i18n.md

```diff
@@ -23,6 +23,10 @@
       - [Formatting Texts](#formatting-texts)
       - [TRANSLATORS Comments](#translators-comments)
       - [Missing Translations](#missing-translations)
+      - [Testing Language](#testing-language)
```

Comment:
Are you sure these are all 4th level under "Marking things for translation"? I would expect them one level up, under Technical details.

Reply:
Ah, right, I overlooked that...